SignalServiceKit/src/Storage/TSRecipient.h

```diff
@@ -1,12 +1,14 @@
 //
-//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
 //
 
+#import "TSYapDatabaseObject.h"
+
 NS_ASSUME_NONNULL_BEGIN
 
 // Some lingering TSRecipient records in the wild causing crashes.
 // This is a stop gap until a proper cleanup happens.
-@interface TSRecipient : NSObject <NSCoding>
+@interface TSRecipient : TSYapDatabaseObject
```

Comment:
@michaelkirk I've expanded the scope of this PR to fix a crash on launch. We now rebuild db extensions that are corrupt on launch.  If a user has a lingering `TSRecipient` instance, this would crash.  It's simpler and safer to make this a `TSYapDatabaseObject` than to use stub implementations which return nil when deserializing. 

FYI this is the `doesNotRecognizeSelector` crash.